### PR TITLE
enable ROSA hcp 4.15

### PR DESCRIPTION
**Pull request by LiangquanLi930**
enable ROSA HCP 4.15, Unified job name

### CODE REVIEW

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-stable.yaml [hunk 1]
```diff
@@ -238,71 +238,74 @@ tests:
...
-  cron: 23 1 4,12,19,26 * *
+    - chain: openshift-e2e-test-hypershift-qe
+    workflow: rosa-aws-sts-hypershift
+- as: aws-rosa-sts-hcp-sector-advanced-prod-critical-f7
+  cron: 56 8 5,14,21,28 * *
   steps:
-    cluster_profile: aws-qe
+    cluster_profile: aws-sd-qe
     env:
       CHANNEL_GROUP: candidate
```
Comment (heliubj18) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-stable.yaml:262]: CLUSTER_SECTOR: canary

Comment (heliubj18) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-stable.yaml:262]: CLUSTER_SECTOR: canary

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-stable.yaml [hunk 2]
```diff
@@ -238,71 +238,74 @@ tests:
...
-  cron: 36 0 4,11,19,27 * *
+    - chain: openshift-e2e-test-hypershift-qe
+    workflow: rosa-aws-sts-hypershift-advanced
+- as: aws-rosa-sts-hcp-sector-private-link-prod-critical-f7
+  cron: 25 17 3,12,19,26 * *
   steps:
     cluster_profile: aws-sd-qe
     env:
       CHANNEL_GROUP: candidate
+      E2E_RUN_TAGS: '@amd64 and @aws-ipi and @rosa and @hypershift-hosted and @critical
```
Comment (heliubj18) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-stable.yaml:279]: CLUSTER_SECTOR: canary

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-stable.yaml [hunk 3]
```diff
@@ -238,71 +238,74 @@ tests:
     test:
     - chain: cucushift-installer-check-cluster-health
     workflow: rosa-aws-sts-advanced
-- as: aws-rosa-sts-private-link-f7
-  cron: 22 20 6,14,21,28 * *
+- as: aws-rosa-sts-hcp-sector-prod-critical-f7
+  cron: 28 6 3,10,17,26 * *
   steps:
-    cluster_profile: aws-qe
+    cluster_profile: aws-sd-qe
```
[the remark this answers is not in the document]
Comment (LiangquanLi930) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-stable.yaml:244]: done

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-stable.yaml (hunk 2 above)
Comment (LiangquanLi930) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-stable.yaml:282]: done

**Comment by LiangquanLi930** [comment]
https://issues.redhat.com/browse/CCO-517

**Comment by LiangquanLi930** [comment]
https://issues.redhat.com/browse/CCO-517

**Comment by LiangquanLi930** [comment]
@liangxia Could you help to review this PR? Thanks

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-stable.yaml [hunk 4]
```diff
@@ -238,71 +238,77 @@ tests:
...
     env:
       CHANNEL_GROUP: candidate
+      CLUSTER_SECTOR: canary
+      E2E_RUN_TAGS: '@amd64 and @aws-ipi and @rosa and @hypershift-hosted and @critical
+        and not @fips'
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-stable.yaml:249]: `not @fips` is no longer needed, more detail in https://github.com/openshift/release/pull/47129 and https://github.com/openshift/release/pull/47158

Comment (LiangquanLi930) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-stable.yaml]: Thanks for your remind, could you help to review it again? Thanks

**Pull request merged**